src/microbe_stage/Microbe.Contact.cs

```diff
@@ -1954,5 +1954,7 @@ public EngulfedObject(IEngulfable @object, Endosome phagosome)
         public (Vector3 Translation, Vector3 Scale, Vector3 EndosomeScale) InitialValuesToLerp { get; set; }
         public Vector3 OriginalScale { get; set; }
         public int OriginalRenderPriority { get; set; }
+        public uint OriginalCollisionLayer { get; set; }
+        public uint OriginalCollisionMask { get; set; }
```

Comment:
Isn't there a potential save incompatibility thing here?
I think these should have default values set to what makes sense for microbes (and a comment added explaining that the values are here for save compatibility).